serverless.yml

```diff
@@ -50,17 +50,20 @@ provider:
     PROJECTS_TABLE: ${self:custom.projectsTable}
     AUTH0_CLIENT_ID: ${file(./secrets.json):AUTH0_CLIENT_ID}
     AUTH0_CLIENT_PUBLIC_KEY: ${file(./public_key)}
-  iamRoleStatements:
-    - Effect: Allow 
-      Action: 
-        - "dynamodb:PutItem"
-        - "dynamodb:GetItem"
-        - "dynamodb:UpdateItem"
-        - "dynamodb:DeleteItem"
-        - "dynamodb:Scan"
-        - "dynamodb:Query"
-      Resource:
-        - "arn:aws:dynamodb:${self:provider.region}:*:table/${self:provider.environment.PROJECTS_TABLE}*"
+    STAGE: ${self:custom.stage.${self:provider.stage}}
```

Comment:
Probably safer to use just `STAGE: ${self:provider.stage}` here instead, because if we ever use the `custom.stage.prod: projects` mapping (line 28), then the environment variable detected in python for prod would be 'projects' and then the calendar url would be `calendar.photonranch.org/projects/...`.